docs/source/hardware/hardware_overview.rst

```diff
@@ -3,129 +3,73 @@
 ******************
 Hardware overview
 ******************
-The Discovery cluster provides you with access to over 24,000 CPU cores and over 200 GPUs. Discovery is connected
+NEU's HPC provides you with access to over 1024 CPU nodes, 50,000 CPU cores, and over 200 GPUs and is connected
```

Comment:
@SerenaCaplins, maybe we can stick with "The Discovery cluster" instead of "NEU's HPC" since most users are familiar with that name. Otherwise, we can also do 
"Northeastern University's high performance computing (HPC) cluster provides you with access to over 1024 CPU nodes, 50,000 CPU cores, and over 200 GPUs. It is connected"